lib/widgets/webview.dart

```diff
@@ -262,6 +276,13 @@ class NestedWebViewController {
     }
     return NavigationActionPolicy.CANCEL;
   }
+
+  void dispose() {
```

Comment:
The method can be placed on line 191. That is, the sequences of methods:
- `final`, `late final`, fields (also getters and setters).
- Overriding methods (`initState`, `didUpdateWidget`, `dispose`), or methods names are likely to override.
- Typed methods (`String _buildContentString`).
- Async methods (`Future<void> _fetch`).
- Void methods (`void _check`).
- `Widget` methods (`Widget _buildBody(BuildContext context)`).
- `build` method.

Reply:
clearly